shopify/standalone-shopify-template/web/routes/signed-in.tsx

```diff
@@ -31,20 +33,20 @@ export default function () {
           web/routes/signed-in.jsx
         </a>
       </Box>
-      <BlockStack className="card-stack">
+      <BlockStack gap="200">
```

Comment:
I think we need to go through `web/components/App.css` and remove all unused classes

Reply:
Done!